packages/core/src/node/dev.ts

```diff
@@ -34,12 +34,21 @@ export async function dev(options: DevOptions): Promise<ServerInstance> {
       false,
       extraBuilderConfig,
     );
+    builder.addPlugins([
```

Comment:
We can also use `rsbuild.onDevCompileDone`

https://rsbuild.dev/api/javascript-api/instance#rsbuildondevcompiledone

Reply:
great

Reply:
The API of Rsbuild is so smooth!